src/CSET/operators/read.py

```diff
@@ -240,6 +243,19 @@ def _deterministic_callback(cube, field, filename):
         )
 
 
+def _um_normalise_callback(cube: iris.cube.Cube, field, filename):
+    """Normalise UM STASH variable long names to LFRic variable names.
+
+    Note UM standard names will remain associated with cubes where different.
+    Long_name will be used consistently in output filename and titles.
+    """
+    # Convert STASH to LFRic variable name
+    if "STASH" in cube.attributes:
+        stash = cube.attributes["STASH"]
+        (name, grid) = STASH_TO_LFRIC.get(str(stash), (None, "a"))
+        cube.long_name = name
```

Comment:
You are correct, I thought I had committed all James' suggestions, but apparently not. Have now (hopefully!) updated + small addition to warning pointing users to relevant dict file.
Agreed there will be additions as we go, but will be important to synchronise with the irislib on repo too rather than let these diverge. If 'someone' can do a major update on both sides sooner than later to capture everything we know we'll need, that would be great.

Reply:
Have committed latest changes, including formatting issue in _stash_to_lfric.py - this seems to have sorted the failing test (git confused me yesterday with that showing as failed, but then passed next time I looked!).

Think all in order now? Thanks for help with first timer.